src/routes/jira/connect/enterprise/jira-connect-enterprise-post.ts

```diff
@@ -89,27 +91,42 @@ export const JiraConnectEnterprisePost = async (
 			jiraHost: jiraHost
 		});
 	} catch (err) {
-		req.log.warn({ err, gheServerURL }, `Couldn't access GHE host`);
-		const codeOrStatus = "" + (err.code || err.response.status);
-
-		req.log.info({ err, gheServerURL }, `Couldn't access GHE host, result of whether skip the check is ${!err.code && err.response?.status}`);
-		if (!err.code && err.response?.status) {
-			//err.code means there's error on the tcp/https connection,
-			//err.status means traffic reach network, but server reject it.
-			//as long as there's no code and a status, means server returns something
-			//so the domain name is reachable, it is just it required some api tokens to be accessible,
-			//and we can bypass this check to allow the manual app creation.
+		const axiosError: AxiosError = (err instanceof GithubClientError) ? err.cause : err;
```

Comment:
Adding typing for the error: had to refactor a bit to make TypeScript happy, but the tests should cover this (already had some + added more)